### PR TITLE
[Continuation of PR 1590] Updated toc_all.md

### PR DESCRIPTION
**Pull request by asm-foysal**
### Change Description ###

Updated the ToC as per the comment https://github.com/Samsung/tizen-docs/pull/1586#pullrequestreview-879897354

Reason: PR 1590 got closed because the source repo accidentally got deleted from the local machine.

### CODE REVIEW

**Comment by mijin85cho** [comment]
Thanks for the PR, @asm-foysal 

It seems this version doesn't include https://github.com/Samsung/tizen-docs/pull/1590/commits/32ea2f06dea6c3fccef33bf2fc29b548854628b8.
Would you please check?

Also, for `VS/VSCode`, please use `Visual Studio/Visual Studio Code Extension`.

**Review by asm-foysal — COMMENTED** [review]
@dkson95 and @mijin85cho I think I got all the changes from PR 1590 in here and fixed the issues mijin85cho pointed out in https://github.com/Samsung/tizen-docs/pull/1590#discussion_r826625853. Can both of you please give it a check and confirm if everything is in order? I'm really sorry about the inconvenience!
P.S.: I have organized the toc structure a bit to make it easier to read in markdown, I hope this isn't a problem...

**Review by dkson95 — CHANGES_REQUESTED** [review]
@asm-foysal First, could you check the comments ?

Also, if possible it'd better to have single `Get Started` instead of `Get Started with Visual Studio` and  `Get Started with Visual Studio Code Extension`.

E.g, https://user-images.githubusercontent.com/9958374/153562379-9df9eb2a-1a0b-4b27-89f2-b21aa275d895.png as @mijin85cho suggested before.

**Review by mijin85cho — COMMENTED** [review]
1.  In Tizen Studio, we have pages for Native/Web.

    However, in case of the tools used in both,
    we put the pages in tizen-studio/common-tools/ folder, and show them in two places with toc_all.md

    For example tizen-studio/common-tools/emulator.md, 
    in toc_all.md, it was written under Native tools and Web tools.

    As the emulator.md is moved to Common Tools,
    these tizen-studio/common-tools/emulator.md under Native & Web should be removed.

    It seems the moved tools are removed from one of Native or Web.

2. As we are going to change toc_all.md,
   I checked the overview pages, to apply the change in toc_all.md

3. >Also, if possible it'd better to have single Get Started instead of Get Started with Visual Studio and Get Started with Visual Studio Code Extension.

    (on chat with @dkson95) It would be good to update the Get Started pages in Native & Web, as like .NET.
    However, to adjust priority of the works,
    for this time we will just follow the current plan.

    After this PR is merged, we will check and merge the Get Started pages.

File: docs/application/toc_all.md [hunk 1]
```diff
@@ -1061,233 +1109,218 @@
...
-##### [Emulator Control Panel](/application/tizen-studio/common-tools/emulator-control-panel.md)
-##### [Extended Emulator Features](/application/tizen-studio/common-tools/emulator-features.md)
-#### Web Simulator
-##### [Overview](/application/tizen-studio/web-tools/web-simulator.md)
-##### [Extended Web Simulator Features](/application/tizen-studio/web-tools/web-simulator-features.md)
-#### [Web Unit Test Tool](/application/tizen-studio/web-tools/web-unit-test-tool.md)
+# Tizen Tools Family 
+
+## Visual Studio Tools for Tizen
+### [Overview](/application/vstools/index.md)
```
[the remark this answers is not in the document]
Comment (asm-foysal) [reply docs/application/toc_all.md:1118]: We should open another PR for this, right?

File: docs/application/toc_all.md [hunk 2]
```diff
@@ -1061,233 +1109,218 @@
...
+##### [Prerequisites](/application/tizen-studio/setup/prerequisites.md)
+##### [Install Tizen Studio](/application/tizen-studio/setup/install-sdk.md)
+##### [Update Tizen Studio](/application/tizen-studio/setup/update-sdk.md)
+##### [Hardware Accelerated Execution Manager](/application/tizen-studio/setup/hardware-accelerated-execution-manager.md)
+
+#### [Configure the Package Manager](/application/tizen-studio/setup/advanced-configuration.md)
+#### [Uninstall Tizen Studio](/application/tizen-studio/setup/uninstall-sdk.md)
+
+### Native Tools
+#### [Overview](/application/tizen-studio/native-tools/index.md)
```
[the remark this answers is not in the document]
Comment (asm-foysal) [reply docs/application/toc_all.md:1153]: Will add it to the new PR.

File: docs/application/toc_all.md [hunk 3]
```diff
@@ -1061,233 +1109,218 @@
...
+##### [Static Analyzer](/application/tizen-studio/native-tools/static-analyzer.md)
+##### [Valgrind](/application/tizen-studio/native-tools/valgrind.md)
+##### [T-trace](/application/tizen-studio/native-tools/t-trace.md)
+##### [Address Sanitizer](/application/tizen-studio/native-tools/address-sanitizer.md)
+##### [Leak Sanitizer](/application/tizen-studio/native-tools/leak-sanitizer.md)
+##### [Code Coverage](/application/tizen-studio/native-tools/code-coverage.md)
+##### [Battery Historian](/application/tizen-studio/native-tools/batt-historian.md)
+
+#### Run and Test Your App
+##### [Overview](/application/tizen-studio/native-tools/running-testing.md)
```
[the remark this answers is not in the document]
Comment (asm-foysal) [reply docs/application/toc_all.md:1211]: Will add this file in the new PR as well.

File: docs/application/toc_all.md [hunk 4]
```diff
@@ -1061,233 +1109,218 @@
...
+
+#### CLI
+##### [Overview](/application/tizen-studio/common-tools/command-line-interface.md)
+##### [Convert Projects for CLI](/application/tizen-studio/native-tools/project-conversion.md)
+##### [RPK Package](/application/tizen-studio/native-tools/rpk-package.md)
+
+#### [Keyboard Shortcuts](/application/tizen-studio/common-tools/keyboard-shortcuts.md)
+
+### Web Tools
+#### [Overview](/application/tizen-studio/web-tools/index.md)
```
[the remark this answers is not in the document]
Comment (asm-foysal) [reply docs/application/toc_all.md:1214]: Will add this file in the new PR as well.

File: docs/application/toc_all.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/toc_all.md:1111]: No we shouldn't. It is related with toc_all.md issue, so we can add that files too in this PR. Also handling the overview files in this PR too is easier to check

File: docs/application/toc_all.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/toc_all.md:1146]: In this PR we are moving Emulator and SDB files away from Tizen Studio Native tools.
So syncing the description with the actual file location is definitely related with this PR also.
You don't need to open a new PR.

Handling this in this PR is proper.

File: docs/application/toc_all.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/toc_all.md:1214]: Please refer https://github.com/Samsung/tizen-docs/pull/1600#discussion_r830695059

File: docs/application/toc_all.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/toc_all.md:1204]: Please refer https://github.com/Samsung/tizen-docs/pull/1600#discussion_r830695059

**Review by asm-foysal — COMMENTED** [review]
@mijin85cho all the changes suggested by you have been incorporated.

**Comment by mijin85cho** [comment]
> @mijin85cho all the changes suggested by you have been incorporated.

I'll check and leave comments.

**Review by dkson95 — APPROVED** [review]
thank you for this update 👍

**Comment by duchuyta** [comment]
Re-open to start trigger build

**Review by mijin85cho — COMMENTED** [review]
@asm-foysal I think most of the updates are done. So many thanks!

I have two suggestion comment, and hope you to check them.


@dkson95 This PR is almost ready.
~but there is an issue in building the stg URL.
(@asm-foysal's last update was not applied to stg.)
I asked iljooo.kim to check.~
build error in stg seems fixed.

~When the stg building is fixed,~ the stg URL can be shared to other members to check.

File: docs/application/toc_all.md [hunk 5]
```diff
@@ -3,21 +3,27 @@
...
-# .NET Application
 
+# .NET Application
 
 ## [Overview](/application/dotnet/index.md)
+
 ## [Learn about .NET](/application/dotnet/learn-about-dotnet.md)
 
-## Get Started
+<!--- vscode-ext/Tizen/dotnet.md and vstools/Tizen/dotnet.md should be merged into dotnet/get-started/first-app.md (PR#1599)
```
Comment (safir-srbd) [inline docs/application/toc_all.md:19]: Reference: https://github.com/Samsung/tizen-docs/pull/1599#issuecomment-1078855364

```suggestion
```

File: docs/application/toc_all.md [hunk 6]
```diff
@@ -3,21 +3,27 @@
...
+## Get Started with Visual Studio Code Extension
+### [.NET application development](/application/vscode-ext/Tizen/dotnet.md)
 
+## Get Started with Visual Studio
+### [.NET application development](/application/vstools/Tizen/dotnet.md)
+--->
```
Comment (safir-srbd) [inline docs/application/toc_all.md:25]: Reference: https://github.com/Samsung/tizen-docs/pull/1599#issuecomment-1078855364

```suggestion
```

**Review by dkson95 — APPROVED** [review]
LGTM 👍

File: docs/application/tizen-studio/native-tools/project-conversion.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
 # Converting Projects for CLI
 
-You can build a project, originally created with Tizen Studio, using the Command Line Interface (CLI).
+You can build a project, originally created with Tizen Studio, using the Command Line Interface (CLI). For more information on CLI, see [Command Line Interface Commands](../common-tools/command-line-interface.md)
```
Comment (mijin85cho) [inline docs/application/tizen-studio/native-tools/project-conversion.md:3]: Maybe a period is needed at the end.

File: docs/application/toc_all.md [hunk 8]
```diff
@@ -223,53 +234,59 @@
...
 ###### [ScrollableBase](/application/dotnet/guides/user-interface/nui/nui-components/ScrollableBase.md)
 ###### [Slider](/application/dotnet/guides/user-interface/nui/nui-components/Slider.md)
 ###### [Switch](/application/dotnet/guides/user-interface/nui/nui-components/Switch.md)
 ###### [Tab](/application/dotnet/guides/user-interface/nui/nui-components/Tab.md)
-###### [TimePicker](/application/dotnet/guides/user-interface/nui/nui-components/TimePicker.md)
-##### [Layer](/application/dotnet/guides/user-interface/nui/layer.md)
+###### [Toast](/application/dotnet/guides/user-interface/nui/nui-components/Toast.md)
+
+##### [Layer](/application/dotnet/guides/user-interface/nui/layer.md)     
+
```
Comment (mijin85cho) [inline docs/application/toc_all.md:254]: Would you please rebase your PR to the latest? It seems PR 1522's update for toc_all.md was reverted here.

File: docs/application/tizen-studio/native-tools/project-conversion.md (hunk 7 above)
Comment (safir-srbd) [inline docs/application/tizen-studio/native-tools/project-conversion.md:3]:
```suggestion
You can build a project, originally created with Tizen Studio, using the Command Line Interface (CLI). For more information on CLI, see [Command Line Interface Commands](../common-tools/command-line-interface.md).
```

File: docs/application/toc_all.md (hunk 6 above)
Comment (mijin85cho) [inline docs/application/toc_all.md:25]: @asm-foysal As you checked on https://github.com/Samsung/tizen-docs/pull/1599#issuecomment-1081330826,
this part should be changed to have similar form with Native and Web.

File: docs/application/toc_all.md [hunk 9]
```diff
@@ -223,53 +234,59 @@
 ###### [Text](/application/dotnet/guides/user-interface/nui/text.md)
 ###### [VideoView](/application/dotnet/guides/user-interface/nui/videoview.md)
 ###### [WebView](/application/dotnet/guides/user-interface/nui/webview.md)
```
Comment (safir-srbd) [inline docs/application/toc_all.md:236]:
```suggestion
###### [WebView](/application/dotnet/guides/user-interface/nui/webview.md)
###### [AlertDialog](/application/dotnet/guides/user-interface/nui/nui-components/AlertDialog.md
```

File: docs/application/toc_all.md [hunk 10]
```diff
@@ -223,53 +234,59 @@
...
-###### [AlertDialog](/application/dotnet/guides/user-interface/nui/nui-components/AlertDialog.md)
 ###### [Button](/application/dotnet/guides/user-interface/nui/nui-components/Button.md)
+###### [CameraView](/application/dotnet/guides/user-interface/nui/nui-components/CameraView.md)
```
Comment (safir-srbd) [inline docs/application/toc_all.md:238]:
```suggestion
```

File: docs/application/toc_all.md [hunk 11]
```diff
@@ -223,53 +234,59 @@
...
 ###### [CheckBox](/application/dotnet/guides/user-interface/nui/nui-components/CheckBox.md)
```
Comment (safir-srbd) [inline docs/application/toc_all.md:239]:
```suggestion
###### [CheckBox](/application/dotnet/guides/user-interface/nui/nui-components/CheckBox.md)
###### [DatePicker](/application/dotnet/guides/user-interface/nui/nui-components/DatePicker.md)
```

File: docs/application/toc_all.md [hunk 12]
```diff
@@ -223,53 +234,59 @@
...
-###### [DatePicker](/application/dotnet/guides/user-interface/nui/nui-components/DatePicker.md)
 ###### [Loading](/application/dotnet/guides/user-interface/nui/nui-components/Loading.md)
 ###### [LottieAnimationView](/application/dotnet/guides/user-interface/nui/nui-components/LottieAnimationView.md)
-###### [Menu](/application/dotnet/guides/user-interface/nui/nui-components/Menu.md)
+###### [Notification](/application/dotnet/guides/user-interface/nui/nui-components/Notification.md)
 ###### [Pagination](/application/dotnet/guides/user-interface/nui/nui-components/Pagination.md)
```
Comment (safir-srbd) [inline docs/application/toc_all.md:243]:
```suggestion
###### [Pagination](/application/dotnet/guides/user-interface/nui/nui-components/Pagination.md)
###### [Picker](/application/dotnet/guides/user-interface/nui/nui-components/Picker.md)
```

File: docs/application/toc_all.md [hunk 13]
```diff
@@ -223,53 +234,59 @@
...
 ###### [Popup](/application/dotnet/guides/user-interface/nui/nui-components/Popup.md)
 ###### [Progress](/application/dotnet/guides/user-interface/nui/nui-components/Progress.md)
 ###### [RadioButton](/application/dotnet/guides/user-interface/nui/nui-components/RadioButton.md)
 ###### [ScrollableBase](/application/dotnet/guides/user-interface/nui/nui-components/ScrollableBase.md)
 ###### [Slider](/application/dotnet/guides/user-interface/nui/nui-components/Slider.md)
 ###### [Switch](/application/dotnet/guides/user-interface/nui/nui-components/Switch.md)
 ###### [Tab](/application/dotnet/guides/user-interface/nui/nui-components/Tab.md)
-###### [TimePicker](/application/dotnet/guides/user-interface/nui/nui-components/TimePicker.md)
-##### [Layer](/application/dotnet/guides/user-interface/nui/layer.md)
+###### [Toast](/application/dotnet/guides/user-interface/nui/nui-components/Toast.md)
```
Comment (safir-srbd) [inline docs/application/toc_all.md:251]:
```suggestion
###### [TimePicker](/application/dotnet/guides/user-interface/nui/nui-components/TimePicker.md)
```

File: docs/application/toc_all.md [hunk 14]
```diff
@@ -223,53 +234,59 @@
...
 ###### [WebView](/application/dotnet/guides/user-interface/nui/webview.md)
-###### [AlertDialog](/application/dotnet/guides/user-interface/nui/nui-components/AlertDialog.md)
 ###### [Button](/application/dotnet/guides/user-interface/nui/nui-components/Button.md)
+###### [CameraView](/application/dotnet/guides/user-interface/nui/nui-components/CameraView.md)
 ###### [CheckBox](/application/dotnet/guides/user-interface/nui/nui-components/CheckBox.md)
-###### [DatePicker](/application/dotnet/guides/user-interface/nui/nui-components/DatePicker.md)
 ###### [Loading](/application/dotnet/guides/user-interface/nui/nui-components/Loading.md)
 ###### [LottieAnimationView](/application/dotnet/guides/user-interface/nui/nui-components/LottieAnimationView.md)
-###### [Menu](/application/dotnet/guides/user-interface/nui/nui-components/Menu.md)
+###### [Notification](/application/dotnet/guides/user-interface/nui/nui-components/Notification.md)
```
Comment (safir-srbd) [inline docs/application/toc_all.md:242]:
```suggestion
###### [Menu](/application/dotnet/guides/user-interface/nui/nui-components/Menu.md)
```

**Review by safir-srbd — COMMENTED** [review]
@mijin85cho @asm-foysal I think these changes should address PR 1522's update for toc_all.md.

**Review by safir-srbd — COMMENTED** [review]
@mijin85cho I also have another question. With PR #1595, #1608, and #1612 being merged into the master, there are three new md files that were added to `toc_all.md`:
- 4.6 Release Notes (`/application/tizen-studio/release-notes/4-6-release-notes.md`)
- VS Code Extension Hybrid Application Development (`/application/vscode-ext/Tizen/native.md`)
- VS Tools Hybrid Application Development (`/application/vstools/Tizen/hybrid.md`)

How should we address them in this PR?

File: docs/application/toc_all.md [hunk 15]
```diff
@@ -223,7 +232,7 @@
 ###### [Text](/application/dotnet/guides/user-interface/nui/text.md)
 ###### [VideoView](/application/dotnet/guides/user-interface/nui/videoview.md)
 ###### [WebView](/application/dotnet/guides/user-interface/nui/webview.md)
-###### [AlertDialog](/application/dotnet/guides/user-interface/nui/nui-components/AlertDialog.md)
+###### [AlertDialog](/application/dotnet/guides/user-interface/nui/nui-components/AlertDialog.md
```
Comment (mijin85cho) [inline docs/application/toc_all.md:235]:
```suggestion
###### [AlertDialog](/application/dotnet/guides/user-interface/nui/nui-components/AlertDialog.md)
```

**Comment by mijin85cho** [comment]
> @mijin85cho I also have another question. With PR #1595, #1608, and #1612 being merged into the master, there are three new md files that were added to `toc_all.md`:
> 
> * 4.6 Release Notes (`/application/tizen-studio/release-notes/4-6-release-notes.md`)
> * VS Code Extension Hybrid Application Development (`/application/vscode-ext/Tizen/native.md`)
> * VS Tools Hybrid Application Development (`/application/vstools/Tizen/hybrid.md`)
> 
> How should we address them in this PR?

1. If git can figure out to where the added pages are inserted, git automatically controls it.
2. However, in this PR, the overall structure of Tizen Tools are changed, so git would not possible to find out how to arrange the updates.
3. So, in each PR, git follows the merged PR, and ask the other PR to resolve the conflict
   **Example :** Release note 4.6  is merged, so git can't figure out where to insert the release note 4.6 in toc_all.md in this PR.
   So it is showing conflict and asking you to resolve it.

   If you do the rebase, you'd see two versions that git can't find out which one to follow

   \<<<<<<< version 1
   version 1 content
   \=======
   version 2 content
   \>>>>>>> version 2

   Check the contents, and update to the latest one.

**Review by safir-srbd — COMMENTED** [review]
@mijin85cho Hope you are well. Thank you for your reply.

 I can see where the new line for the Release Notes 4.6 would fit into the new format for `toc_all.md`, but under the new structure, I can't seem to find an appropriate place to put in the two lines for the new articles for VS Code Extension Hybrid Application Development and VS Tools Hybrid Application Development.

Could you suggest an appropriate place where these two lines can be placed?

**Comment by mijin85cho** [comment]
> @mijin85cho Hope you are well. Thank you for your reply.
> 
> I can see where the new line for the Release Notes 4.6 would fit into the new format for `toc_all.md`, but under the new structure, I can't seem to find an appropriate place to put in the two lines for the new articles for VS Code Extension Hybrid Application Development and VS Tools Hybrid Application Development.
> 
> Could you suggest an appropriate place where these two lines can be placed?

As the Hybrid App doesn't belong to .NET nor Native,
I recommend you to place those pages where each author placed it.
Under vscode-ext/Tizen and vstools/Tizen****

**Review by safir-srbd — COMMENTED** [review]
> > @mijin85cho Hope you are well. Thank you for your reply.
> > I can see where the new line for the Release Notes 4.6 would fit into the new format for `toc_all.md`, but under the new structure, I can't seem to find an appropriate place to put in the two lines for the new articles for VS Code Extension Hybrid Application Development and VS Tools Hybrid Application Development.
> > Could you suggest an appropriate place where these two lines can be placed?
> 
> As the Hybrid App doesn't belong to .NET nor Native, I recommend you to place those pages where each author placed it. Under vscode-ext/Tizen and vstools/Tizen****

@mijin85cho So it should be something like this below, right?

`# Visual Studio`
`## Tizen`
`### [Hybrid application development](/application/vstools/Tizen/hybrid.md)`

`#  Visual Studio Code Extension for Tizen`
`## Tizen`
`### [Hybrid application development](/application/vscode-ext/Tizen/hybrid.md)`

Although in the new structure, this might not fit well, as the VS Tools and VS Code Extension articles have been divided under dotnet, native, and web. Do let me know your thoughts.

**Comment by mijin85cho** [comment]
> > > @mijin85cho Hope you are well. Thank you for your reply.
> > > I can see where the new line for the Release Notes 4.6 would fit into the new format for `toc_all.md`, but under the new structure, I can't seem to find an appropriate place to put in the two lines for the new articles for VS Code Extension Hybrid Application Development and VS Tools Hybrid Application Development.
> > > Could you suggest an appropriate place where these two lines can be placed?
> > 
> > 
> > As the Hybrid App doesn't belong to .NET nor Native, I recommend you to place those pages where each author placed it. Under vscode-ext/Tizen and vstools/Tizen****
> 
> @mijin85cho So it should be something like this below, right?
> 
> `# Visual Studio` `## Tizen` `### [Hybrid application development](/application/vstools/Tizen/hybrid.md)`
> 
> `# Visual Studio Code Extension for Tizen` `## Tizen` `### [Hybrid application development](/application/vscode-ext/Tizen/hybrid.md)`
> 
> Although in the new structure, this might not fit well, as the VS Tools and VS Code Extension articles have been divided under dotnet, native, and web. Do let me know your thoughts.


Yes, you're right. It will looks like below.

```
# Tizen Tools Family 

## Visual Studio Tools for Tizen
### [Overview](/application/vstools/index.md)
### [Installing Visual Studio Tools for Tizen](/application/vstools/install.md)

### Tools
#### [Project Wizard](/application/vstools/tools/project-wizard.md)
...

### Tizen <-- Target location
#### [Hybrid application development](/application/vstools/Tizen/hybrid.md)

## Visual Studio Code Extension for Tizen
### [Prerequisites](/application/vscode-ext/index.md)
### Tizen <-- Target location
#### [Hybrid application development](/application/vscode-ext/Tizen/hybrid.md)
### [Tizen Web](/application/vscode-ext/web.md)

```

Maybe we can find better location later, but for now, let's proceed with current structure.

File: docs/application/toc_all.md [hunk 16]
```diff
@@ -1061,233 +1103,207 @@
...
+#### [Platform Development Tools](/application/tizen-studio/platform-tools/platform-tools.md)
+
+### [Configurable SDK](/application/tizen-studio/configurable-sdk/configurable-sdk.md)
+### Extension SDK
+#### [Overview](/application/tizen-studio/extension-sdk/overview.md)
+
+### Tizen RT
+#### [Overview](/application/tizen-studio/rt-ide/overview.md)
+
+### Release Notes
```
Comment (safir-srbd) [inline docs/application/toc_all.md:1268]:
```suggestion
### Release Notes
#### [4.6 (Mar 30, 2022)](/application/tizen-studio/release-notes/4-6-release-notes.md)
```

File: docs/application/toc_all.md [hunk 17]
```diff
@@ -1061,233 +1103,207 @@
...
+##### [Additional Manifest Elements](/application/vstools/tools/manifest-elements.md)
+
+#### [Emulator Manager](/application/vstools/tools/emulator-manager.md)
+#### [Emulator Control Panel](/application/vstools/tools/emulator-control-panel.md)
+#### [API and Privilege Checker](/application/vstools/tools/api-privilege-checker.md)
+#### [Log Viewer](/application/vstools/tools/log-viewer.md)
+#### [Profiler](/application/vstools/tools/profiler-user-manual.md)
+#### [Memory Profiler](/application/vstools/tools/memory-profiler-user-manual.md)
+#### [Command Line Interface](/application/vstools/tools/dotnet-cli-ext.md)
+
```
Comment (safir-srbd) [inline docs/application/toc_all.md:1129]:
```suggestion

### Tizen
#### [Hybrid application development](/application/vstools/Tizen/hybrid.md)

```

File: docs/application/toc_all.md [hunk 18]
```diff
@@ -1061,233 +1103,207 @@
...
+## Visual Studio Code Extension for Tizen
+### [Prerequisites](/application/vscode-ext/index.md)
```
Comment (safir-srbd) [inline docs/application/toc_all.md:1131]:
```suggestion
### [Prerequisites](/application/vscode-ext/index.md)
### Tizen
#### [Hybrid application development](/application/vscode-ext/Tizen/hybrid.md)
```

**Review by safir-srbd — COMMENTED** [review]
@mijin85cho I think these suggestions should address the addition of new articles into the `toc_all.md`. Check if this is okay whenever you can.

File: docs/application/toc_all.md (hunk 16 above)
Comment (mijin85cho) [reply docs/application/toc_all.md:1268]: I agree

File: docs/application/toc_all.md (hunk 17 above)
Comment (mijin85cho) [reply docs/application/toc_all.md:1129]: I agree

File: docs/application/toc_all.md (hunk 18 above)
Comment (mijin85cho) [reply docs/application/toc_all.md:1131]: I agree

**Comment by mijin85cho** [comment]
@safir-srbd I agree that locations you pointed are where I thought. With current, please resolve the conflict. As you can see in the bottom of this page, we can't merge this PR as toc_all.md is conflicted.

**Review by safir-srbd — APPROVED** [review]

**Pull request merged**